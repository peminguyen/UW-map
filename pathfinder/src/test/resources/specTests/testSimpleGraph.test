CreateGraph graph

AddNode graph A
AddNode graph B

AddEdge graph A B 3

FindPath graph A B