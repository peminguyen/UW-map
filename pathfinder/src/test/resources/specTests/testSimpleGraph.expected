created graph graph

added node A to graph
added node B to graph

added edge 3.000 from A to B in graph

path from A to B:
A to B with weight 3.000
total cost: 3.000